RecoHGCal/TICL/interface/TracksterInferenceByCNNv4.h

```diff
@@ -0,0 +1,42 @@
+#ifndef RecoHGCal_TICL_TracksterInferenceByCNNv4_H__
+#define RecoHGCal_TICL_TracksterInferenceByCNNv4_H__
+
+#include "PhysicsTools/ONNXRuntime/interface/ONNXRuntime.h"
+#include "RecoHGCal/TICL/interface/TracksterInferenceAlgoBase.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "PhysicsTools/TensorFlow/interface/TensorFlow.h"
+#include "RecoLocalCalo/HGCalRecAlgos/interface/RecHitTools.h"
+
+namespace ticl {
+  using namespace cms::Ort;
+
+  class TracksterInferenceByCNNv4 : public TracksterInferenceAlgoBase {
+  public:
+    explicit TracksterInferenceByCNNv4(const edm::ParameterSet& conf);
+    void inputData(const std::vector<reco::CaloCluster>& layerClusters, std::vector<Trackster>& tracksters) override;
+    void runInference(std::vector<Trackster>& tracksters) override;
+
+    static void fillPSetDescription(edm::ParameterSetDescription& iDesc);
+
+  private:
+    const cms::Ort::ONNXRuntime* onnxSession_;
+
+    const std::string modelPath_;
+    const std::vector<std::string> inputNames_;
+    const std::vector<std::string> outputNames_;
+    const float eidMinClusterEnergy_;
+    const int eidNLayers_;
+    const int eidNClusters_;
+    static constexpr int eidNFeatures_ = 3;
+    int doPID_;
+    int doRegression_;
+
+    hgcal::RecHitTools rhtools_;
+    std::vector<std::vector<int64_t>> input_shapes;
```

Comment:
Use the same pattern used above, i.e., mark private members with a trailing `_`.
Here and in the following 3 lines.